src/Backend/Modules/MediaLibrary/Ajax/MediaFolderMove.php

```diff
@@ -83,7 +84,7 @@ private function getTypeOfDrop(): string
             throw new AjaxExitException('no type provided');
         }
 
-        if (!in_array($typeOfDrop, ['before', 'after', 'inside'], true)) {
+        if (!in_array($typeOfDrop, Model::POSSIBLE_TYPES_OF_DROP, true)) {
```

Comment:
I don't find this readable, in my opinion, it would make more sense to list the possible drop-types here in the method.

Reply:
It might be less readable, but the good thing about this is that if you change it in your model, it's automatically changed everywhere. Imo, this is better (less duplication, less possible future bugs) than listing them up here.

Reply:
I do not agree, I prefer readability over developer-laziness.
If you don't want to have code-duplication, the proper solution would be to extract the check into a separate method which checks if the type of drop is allowed.